src/component/playground/PlaygroundResultsTable/PlaygroundResultsTable.tsx

```diff
@@ -0,0 +1,227 @@
+import { LinkCell } from '../../common/Table/cells/LinkCell/LinkCell';
+import { FeatureStatusCell } from './FeatureStatusCell/FeatureStatusCell';
+import { useSearchParams } from 'react-router-dom';
+import { useSearch } from '../../../hooks/useSearch';
+import { createLocalStorage } from '../../../utils/createLocalStorage';
+import { PlaygroundFeatureSchema } from '../../../openapi';
+
+const defaultSort: SortingRule<string> = { id: 'name' };
+const { value, setValue } = createLocalStorage(
+    'PlaygroundResultsTable:v1',
+    defaultSort
+);
+
+interface IPlaygroundResultsTableProps {
+    features: PlaygroundFeatureSchema[];
+    loading: boolean;
+}
+
+export const PlaygroundResultsTable = ({
+    features,
+    loading,
+}: IPlaygroundResultsTableProps) => {
+    const [searchParams, setSearchParams] = useSearchParams();
+
+    const [searchValue, setSearchValue] = useState(
+        searchParams.get('search') || ''
+    );
+
+    const {
+        data: searchedData,
+        getSearchText,
+        getSearchContext,
+    } = useSearch(COLUMNS, searchValue, features);
+
+    const data = useMemo(() => {
+        return loading
+            ? Array(5).fill({
+                  name: 'Feature name',
+                  project: 'Feature Project',
+                  variant: 'Feature variant',
+                  enabled: 'Feature state',
+              })
+            : searchedData;
+    }, [searchedData, loading]);
+
+    const [initialState] = useState(() => ({
+        sortBy: [
+            {
+                id: searchParams.get('sort') || value.id,
+                desc: searchParams.has('order')
+                    ? searchParams.get('order') === 'desc'
+                    : value.desc,
+            },
+        ],
+        hiddenColumns: [],
+    }));
+
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        state: { sortBy },
+        rows,
+        prepareRow,
+    } = useTable(
+        {
+            initialState,
+            columns: COLUMNS as any,
+            data: data as any,
+            sortTypes,
+            autoResetGlobalFilter: false,
+            autoResetSortBy: false,
+            disableSortRemove: true,
+            defaultColumn: {
+                Cell: HighlightCell,
+            },
+        },
+        useGlobalFilter,
+        useSortBy
+    );
+
+    useEffect(() => {
+        if (loading) {
+            return;
+        }
+        const tableState: Record<string, string> = {};
+        tableState.sort = sortBy[0].id;
+        if (sortBy[0].desc) {
+            tableState.order = 'desc';
+        }
+        if (searchValue) {
+            tableState.search = searchValue;
+        }
+
+        setSearchParams(tableState, {
+            replace: true,
+        });
+        setValue({ id: sortBy[0].id, desc: sortBy[0].desc || false });
+    }, [loading, sortBy, searchValue]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Would prefer that we don't use this annotation, what's missing? setValue should be stable if it's a react setter.